### ISSUE TITLE
[SUGGESTION] Support for key icons?

### ISSUE THREAD

**Issue by Clark919** [2394425147/LC_BetterMonitor#3]
Something I really love about the implementation of this mod is the custom icon implementation for particular scrap items. While I don't agree that for balance reasons the terminal operator should know what particular pieces of valuable scrap are, I do quite like the concept of the map showing a little key icon wherever a key is presently. This way you'd get a dynamic where the terminal operator can't see locked doors, but can see keys, and your players can't find keys but know of locked doors. Sounds like a fun opportunity for teamwork, and something like that could be an interesting addition to this mod if possible.

**Comment by Fl4sHNoX** [comment]
I agree with both statements made and wanted to give a heads-up that there's a mod that came out that aims to do what you're looking for aptly named "[KeysOnTheMap](https://github.com/vibeside/KeysOnTheMap)".

Here's the Thunderstore link as well:
https://thunderstore.io/c/lethal-company/p/the_croods/KeysOnMap/
